src/library/Uno.Cupertino/Styles/Controls/ComboBox.xaml

```diff
@@ -117,19 +116,13 @@
 
 						<!-- ContentPresenter-->
 						<ContentPresenter x:Name="ContentPresenter"
+										  Content="{TemplateBinding Content}"
+										  ContentTemplate="{TemplateBinding ContentTemplate}"
+										  ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}"
 										  ContentTransitions="{TemplateBinding ContentTransitions}"
-										  Content="{TemplateBinding Content}">
-							<ContentPresenter.ContentTemplate>
-								<DataTemplate>
-									<Grid>
-										<TextBlock Text="{Binding}"
-												   Style="{ThemeResource CupertinoFootnote}"
```

Comment:
the content inside the ContentTemplate is not needed anymore (using CupertinoFootnote) ?

Reply:
that exactly what we want to avoid, an hardcode content-template that is not overridable
re: CupertinoFootnote, we will continue in steve's comment